common/models/templates/test/test-trie-traversal.js

```diff
@@ -35,6 +35,10 @@ describe('Trie traversal abstractions', function() {
   it('traversal with simple internal nodes', function() {
     var model = new TrieModel(jsonFixture('tries/english-1000'));
 
+    // Prob:  entry weight / total weight
+    // "the" is the highest-weighted word in the fixture.
+    const PROB_OF_THE = 1000 / 500500;
```

Comment:
Can we move this and `PROB_OF_TROUBLE` outside the functions so we can use them throughout?

Reply:
We only use them within this single unit-test definition.